scripts/collect_workloads.py

```
@@ -1015,6 +1026,111 @@ def _pad_to_tokens(msgs: list[dict], approx_tokens: int) -> list[dict]:
+            "--definitions",
+            def_name,
+            "--solutions",
+            "baseline",
+            "--save-results",
+            "--warmup-runs",
+            "3",
+            "--iterations",
+            "20",
+        ]
+        print(f"  Running: {' '.join(cmd)}")
+        result = subprocess.run(cmd, capture_output=True, text=True)
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)